### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by shadeofblue**

### CODE REVIEW

File: requestor-tutorials/dapps/hello-world-dapp.md [hunk 1]
```diff
@@ -305,9 +317,18 @@ There's one more piece of data that we'll need to run our application. It's the
 Unless you want to customize your set-up, it'll be easiest to just use the default that comes with the `dapp-runner`, which you can get with:
 
 ```shell
+
```
Comment (approxit) [inline requestor-tutorials/dapps/hello-world-dapp.md:320]: This empty line is not needed.

**Review by approxit — APPROVED** [review]
After changes from `hello_world` to `hello_golem` should `.md` file be changed too?

**Review by lucekdudek — APPROVED** [review]

**Comment by shadeofblue** [comment]
> After changes from `hello_world` to `hello_golem` should `.md` file be changed too?

dunno, I think it's not really 1:1 connected...

File: requestor-tutorials/dapps/hello-world-dapp.md (hunk 1 above)
Comment (shadeofblue) [inline requestor-tutorials/dapps/hello-world-dapp.md:320]:
```suggestion
```

**Pull request merged**